contrib/docs/tutorials/quickstart-app-plugin/ExampleComponent.md

```diff
@@ -34,7 +34,7 @@ const ExampleComponent = () => {
       </Header>
       <Content>
         <ContentHeader title="Plugin title">
-          <SupportButton>A description of your plugin goes here.</SupportButton>
+          <SupportButton title={'A description of your plugin goes here.'} />
```

Comment:
It's not required to wrap a straight string in a `{`

```suggestion
          <SupportButton title="A description of your plugin goes here." />
```

Same goes for all other instances.